obj_parser/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "obj_parser"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[lib]
+crate-type = ["cdylib"]
+
+[dependencies]
+cimvr_common = { path = "../../common" }
```

Comment:
These two deps should be updated to reflect the new location of the crate! See `cargo check`.